tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,72 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, ".profile")
+    _button_logout = (By.CSS_SELECTOR, ".btn")
+    _dashboard_navigator_locator = (By.ID, 'dashboardNav')
+    _dashboard_login_locator = (By.ID, 'login')
+    _popup_intro = (By.ID, 'intro')
+    _button_popup_intro = (By.ID, 'intro-submit')
+    _assigned_icon = (By.CSS_SELECTOR, '#dashboardNav .assigned')
+    _flagged_icon = (By.CSS_SELECTOR, '#dashboardNav .flagged')
+    _filed_icon = (By.CSS_SELECTOR, '#dashboardNav .filed')
+    _search_bar = (By.ID, 'searchLink')
+    _button_cancel_search = (By.LINK_TEXT, 'Cancel')    
+    def login (self, username, password):
+        username_element = self.marionette.find_element(*self._given_username)
+        username_element.tap()
+        username_element.send_keys(username)
+        password_element = self.marionette.find_element(*self._given_password)
+        password_element.tap()
+        password_element.send_keys(password)
+        button_element = self.marionette.find_element(*self._button_login)
+        button_element.tap()
+
+    def logout(self):
+        profile_element = self.marionette.find_element(*self._profile_icon)
+        profile_element.tap()
+        logout_element = self.marionette.find_element(*self._button_logout)
+        logout_element.tap()
+
+    def followBugs(self):
+        print'function started'
+        flagged_element = self.marionette.find_element(*self._flagged_icon)
```

Comment:
Nit: You don't need to store the element in a variable if you just call one function on it. So here, we can have: ```python  self.marionette.find_element(*self._flagged_icon).tap()```